### PR TITLE
adding tag action to both emr and dynamodb

### PR DESCRIPTION
**Pull request by alfredgamulo**
also removing stray .DS_Store and including it in gitignore

### CODE REVIEW

**Comment by alfredgamulo** [comment]
@kapilt could you help me to understand why the ci is not reporting back coverage?

**Comment by alfredgamulo** [comment]
test this please

File: c7n/resources/dynamodb.py [hunk 1]
```diff
@@ -120,7 +120,36 @@ def process_resource_set(self, tables, tags):
...
+    """
+
+    permissions = ('dynamodb:TagResource',)
+    batch_size = 1
+
+    def process_resource_set(self, tables, tags):
+        client = local_session(self.manager.session_factory).client('dynamodb')
+        for t in tables:
+            arn = t['TableArn']
+            client.tag_resource(ResourceArn=arn, Tags=tags)
```
Comment (kapilt) [inline c7n/resources/dynamodb.py]: should probably using retry with this, but okay for now

File: tests/test_emr.py [hunk 2]
```diff
@@ -98,6 +98,28 @@ def test_emr_mark(self):
...
+                'resource': 'emr',
+                'filters': [{'tag:first_tag': 'first'}],
+                'actions': [{
+                    'type': 'tag',
+                    'tags': {'new_tag_key': 'new_tag_value'}
+                }]
+            },
+            session_factory=session_factory)
+        resources = p.run()
+        new_tags = resources[0]['Tags']
```
Comment (kapilt) [inline tests/test_emr.py]: we don't modify refetch/resources with actions, so this wouldn't work on a fresh record, without another policy run.

**Pull request merged**